Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM nfcore/base
+
+LABEL description="Docker image containing all requirements for coproID pipeline"
+COPY environment.yml /
+RUN conda env create -f /environment.yml && conda clean -a
+ENV PATH /opt/conda/envs/nf-core-coproid-1.0dev/bin:$PATH
+RUN Rscript -e "install.packages('markdown', dependencies=TRUE, repos='http://cloud.r-project.org/')"
```

Comment:
Can you install `markdown` this within the conda environment. If its not in `environment.yml` then the pipeline will fail when run with `-profile conda`.

I didnt have to put it in my `environment.yml` because it was probably a dependency for another package.
https://github.com/nf-core/atacseq/blob/master/environment.yml

Reply:
Fixed with #7